src/backends/postgresql/standard-use-type.cpp

```diff
@@ -159,7 +160,25 @@ void postgresql_standard_use_type_backend::pre_use(indicator const * ind)
                 snprintf(buf_, bufSize, "%lu", bbe->oid_);
             }
             break;
-
+        case x_xmltype:
+            {
+                xml_type * xml = static_cast<xml_type *>(data_);
+                std::string * s = &(xml->value);
+                
+                buf_ = new char[s->size() + 1];
+                std::strcpy(buf_, s->c_str());
+            }
+            break;
+        case x_longstring:
+            {
+                long_string * ls = static_cast<long_string *>(data_);
+                std::string * s = &(ls->value);
+                
+                buf_ = new char[s->size() + 1];
+                std::strcpy(buf_, s->c_str());
```

Comment:
Are we sure that there can be no embedded `NUL` bytes in long strings? We probably are, but OTOH using `memcpy()` instead of `strcpy()` would work in any case and wouldn't really be any more difficult in this case...